README.md

```diff
@@ -101,12 +101,22 @@ if err := conn.Close(); err != nil {
 ```
 
 ### To Create Topics
+By default kafka has the `KAFKA_AUTO_CREATE_TOPICS_ENABLE='true'`. If this value is set to `'true'` then topics will be created as a side effect of `kafka.DialLeader` like so:
```

Comment:
`KAFKA_AUTO_CREATE_TOPICS_ENABLE` is an environment variable of the docker image we use, I don't think it's a standard variable recognized by kafka itself (tho I could be wrong).

Maybe refer to this feature more literally as "automatic topic creation" ?

Reply:
This also does not allow passing options for the topic

Reply:
@achille-roussel good point, in kafka downloaded from [here](https://kafka.apache.org/quickstart) `auto.create.topics.enable` is a config setting in `config/server.properties`, I can update the docs to reflect the two different representations for the same setting

Reply:
addressed by https://github.com/segmentio/kafka-go/pull/546/commits/fd254c74da45417c13b65fa6ce127969e582eacf